jujugui/static/gui/src/app/app.js

```diff
@@ -1588,7 +1599,106 @@ YUI.add('juju-gui', function(Y) {
       @method _handleNewModel
     */
     _handleNewModel: function() {
-      this.switchEnv(null, null, null, null, false, true);
+      this.switchEnv();
+    },
+
+    /**
+      Handle the request to display the user entity state.
+
+      @method _handleUserEntity
+      @param {Object} state - The application state.
+      @param {Function} next - Run the next route handler, if any.
+    */
+    _handleUserEntity: function(state, next) {
+      const userPath = state.user;
+      const handleEntity = this._handleUserEntityPath.bind(
+        this, state, next, userPath);
+      // Attempt to handle an exisitng path.
+      const existing = handleEntity();
+      if (!existing) {
```

Comment:
`existing` here doesn't really mean anything to me..
```
if (!handleEntity()) {
```
imho reads a little better and you never use `existing` again anyways.